spec/requests/entrypoint_spec.rb

```diff
@@ -74,7 +74,10 @@
     )
 
     # This test will fail if you have the UI checked out and built with yarn
```

Comment:
![](https://i.kym-cdn.com/entries/icons/mobile/000/007/423/RageFace.jpg)

It still behaves differently when yarn is being used, because the asset manifest exists...

Reply:
Yet, the "UI is available" version of the test looks suspiciously the same.

But I'll believe you :)